src/env/dev/terraform.tfvars

```diff
@@ -183,7 +183,7 @@ eventhubs = [
     name              = "nodo-dei-pagamenti-re"
     partitions        = 1 # in PROD shall be changed
     message_retention = 1 # in PROD shall be changed
-    consumers         = ["nodo-dei-pagamenti-rx1"]
+    consumers         = ["nodo-dei-pagamenti-rx1", "nodo-dei-pagamenti-rx2"]
```

Comment:
maybe better use owners names, ie pdnd, operations.
check nodo-dei-pagamenti-re and nodo-dei-pagamenti-log

Reply:
yes , according to PDND colleague too, i changed all consumers group names